tensorflow/training/docker/2.8/py3/Dockerfile.cpu

```diff
@@ -173,6 +285,8 @@ RUN HOME_DIR=/root \
    && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} ${PYTHON} \
    && rm -rf ${HOME_DIR}/oss_compliance*
 
+
 RUN curl https://aws-dlc-licenses.s3.amazonaws.com/tensorflow-2.8/license.txt -o /license.txt
```

Comment:
Why is this being done twice? Once [above](https://github.com/aws/deep-learning-containers/pull/1685/files#diff-5c03243703718c5b702996323223a1bebd1aaadac44a9f73158b130cbbf86815R154) and once here?